src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -89,11 +89,59 @@
  * </p>
  * <pre>
  * &#64;Deprecated
- * public static final int MY_CONST = 123456; // no violation
+ * public static final int MY_CONST = 13; // ok
  *
  * &#47;** This javadoc is missing deprecated tag. *&#47;
  * &#64;Deprecated
  * public static final int COUNTER = 10; // violation
+ *
+ * &#47;**
+ * * &#64;deprecated
+ * * &lt;p&gt;&lt;/p&gt;
+ * *&#47;
+ * &#64;Deprecated
+ * public static final int NUM = 123456; // ok
+ *
+ * &#47;**
+ * * &#64;deprecated
```

Comment:
here and below asterisk should have indent +1

Reply:
@strkkk I am not getting this one .Could you please show me with an example (Really sorry for my inefficiency).

![image](https://user-images.githubusercontent.com/42171790/111883625-814f3c80-89e2-11eb-9d89-2714d6cad847.png)

I  tried to imitate this

Reply:
![image](https://user-images.githubusercontent.com/812984/111890017-f569f000-89a2-11eb-8bac-a523f1d12b07.png)

please add one leading space for 3rd and 2nd line of javadoc

Reply:
Done